src/www/ui_components/app-root.js

```diff
@@ -372,7 +372,7 @@ export class AppRoot extends mixinBehaviors([AppLocalizeBehavior], PolymerElemen
               [[localize('about-page-title')]]
             </paper-item>
             <paper-item name="help">
-              <a href="https://s3.amazonaws.com/outline-vpn/index.html#/support" id="helpAnchor" hidden=""></a>
+              <a href="https://getoutline.org/how-it-works" id="helpAnchor" hidden=""></a>
```

Comment:
Let's switch it to https://support.getoutline.org instead, since that's a better destination for a user who's looking for help.

Reply:
Changed the link to the new one, sorry for long reply.

Reply:
@fortuna / @maddyhof , just wonder if it's ok to switch from `s3.amazonaws.com` to `getoutline.org` because the latter one is easier to get blocked.

Reply:
We don't have an S3 mirror of the Support site, and AFAIK it hasn't gotten blocked, even when getoutline.org has (but correct me if I'm wrong).